synapse/config/emailconfig.py

```diff
@@ -158,9 +159,10 @@ def read_config(self, config, **kwargs):
 
             if config.get("public_baseurl") is None:
                 raise RuntimeError(
-                    "email.password_reset_behaviour is set to 'local' but no "
-                    "public_baseurl is set. This is necessary to generate password "
-                    "reset links"
+                    "Password resets emails will be sent from this homeserver "
```

Comment:
rather than duplicating the wording above, with slight diferences, how about:

at line 115:

```
    missing.append("email." + k)
```

then, before line 117:

```
if config.get("public_baseurl") is None:
    missing.append('public_base_url')
```